### PR TITLE
update OKR progress

### PR DESCRIPTION
**Pull request by waligob**

### CODE REVIEW

File: handbook/okrs/CY22-Q2/team.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ _Key Result_: each of the following business categories has at least one data se
 _Objective_: Ensure data platform adoption, utilization, and socialization by providing data engineers and analysts hands-on experience with the platform technologies.
 
 #### O3KR1
-_Key Result_: Each member of the team performs data engineering work that contributes to a Q2 Product Showcase presentation [0/3]
+_Key Result_: Each member of the team performs data engineering work that contributes to a Q2 Product Showcase presentation [1/3]
```
Comment (waligob) [inline handbook/okrs/CY22-Q2/team.md:27]: https://philly.slack.com/archives/C02S39GL47R/p1651085416850779?thread_ts=1651084748.109349&cid=C02S39GL47R

File: handbook/okrs/CY22-Q2/team.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ _Key Result_: each of the following business categories has at least one data se
...
 
 #### O3KR2
 _Key Result_: 
-Key Result 2: At least 4 members of the Analytics team have submitted and merged at least 2 pull requests each for changes to the Inquirer's dbt codebase. [0/8]
+Key Result 2: At least 4 members of the Analytics team have submitted and merged at least 2 pull requests each for changes to the Inquirer's dbt codebase. [1/8]
```
Comment (waligob) [inline handbook/okrs/CY22-Q2/team.md:31]: https://github.com/phillymedia/inquirer-dbt/pull/94

**Review by riordan — APPROVED** [review]
Heck yeah! Love to see it

File: handbook/okrs/CY22-Q2/waligob.md [hunk 3]
```diff
@@ -7,13 +7,13 @@ _Objective_:  Lead the technical design and implementation of an end-to-end mode
 _Key Result_: The data engineering and analytics teams have write access to a data catalog tool that we’ve selected to fulfill the organization’s data discovery needs. [0/1]
 
 #### O1KR2 
-Any component of our cloud infrastructure can be deployed via a push-button process [1.5/4]
+Any component of our cloud infrastructure can be deployed via a push-button process [2.5/4]
```
Comment (waligob) [inline handbook/okrs/CY22-Q2/waligob.md:10]: https://github.com/phillymedia/inq-warehouse-terraform/pull/2

File: handbook/okrs/CY22-Q2/team.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ _Objective_: Design, implement, maintain, and enhance a modern data platform tha
 _Key Result_: At least one new end-to-end use case is operationalized on the data platform, including data capture, modeling, transformation, orchestration, and discovery. [1/5]
 
 #### O1KR2 
-All 3 codebases and 4 cloud infrastructure services that currently comprise the data platform are tracked via source control, and are deployed via CI/CD. [4/14]
+All 3 codebases and 4 cloud infrastructure services that currently comprise the data platform are tracked via source control, and are deployed via CI/CD. [7/14]
```
Comment (waligob) [inline handbook/okrs/CY22-Q2/team.md:10]: https://github.com/phillymedia/inq-warehouse-terraform/pull/2

**Pull request merged**